### DISCUSSION TITLE
Good documentations about snell (how does snell works?)

### DISCUSSION THREAD

**Discussion by iamtrazy** [icpz/open-snell#7] [Ideas]
hi, Im just curious about how does snell works, what kind of encryption protocols does snell use? in which conditions does this perform better than something like vless , vmess , trojan or ssr ? does this perform better on arm platforms or x86_64 platforms , how does snell handles UDP?. I wasn't able to find any good documentations about this project in english.

**Comment by icpz** [comment]
This project implements the https://github.com/surge-networks/snell project from scratch. But the original project has been archived and won't maintain anymore. I personally don't suggest you to waste any time on this project.

**Reply by iamtrazy** [reply]
thats too bad.what would u think will be the best alternative, i just want to make my traffic looks like its from facebook or whatsapp. i was able to do that with snell using tls obfuscation and setting the SNI as facebook.com , it was very fast and had better latency than vless. i tried shadowsocks with simple obfs , but it didnt support UDP. sorry to bother you.

**Reply by icpz** [reply]
I think trojan is your best alternative, which I've also migrated to. The only short of trojan is the first packet latency.
